snapcraft.yaml

```diff
@@ -34,15 +38,11 @@ parts:
     source-type: git
     source-tag: '3.38.4'
     plugin: meson
-    meson-parameters: [--prefix=/usr]
-    organize:
-      snap/gnome-hitori/current/usr: usr
-    override-pull: |
-      snapcraftctl pull
-      snapcraftctl set-version $(git describe --tags --abbrev=10)
+    parse-info: [usr/share/metainfo/org.gnome.Hitori.appdata.xml]
+    meson-parameters:
+      - --prefix=/usr
     override-build: |
-      sed -i.bak -e 's|=org.gnome.Hitori$|=${SNAP}/meta/gui/org.gnome.Hitori.svg|g' $SNAPCRAFT_PART_SRC/data/org.gnome.Hitori.desktop.in
-      snapcraftctl build
-      mkdir -p $SNAPCRAFT_PART_INSTALL/meta/gui/
-      cp ../src/data/icons/scalable/org.gnome.Hitori.svg $SNAPCRAFT_PART_INSTALL/meta/gui/
-      cp data/org.gnome.Hitori.desktop $SNAPCRAFT_PART_INSTALL/meta/gui/
+      sed -i.bak -e 's|=org.gnome.Hitori$|=${SNAP}/meta/gui/org.gnome.Hitori.svg|g' $CRAFT_PART_SRC/data/org.gnome.Hitori.desktop.in
+      craftctl default
+      mkdir -p $CRAFT_PART_INSTALL/meta/gui/
+      cp ../src/data/icons/scalable/org.gnome.Hitori.svg $CRAFT_PART_INSTALL/meta/gui/
```

Comment:
use $CRAFT_PART_SRC instead of ..src/data/blah

Reply:
Done.